php/src/Google/Protobuf/Internal/DescriptorProto.php

```diff
@@ -12,66 +12,62 @@
 use Google\Protobuf\Internal\GPBUtil;
 
 /**
- * <pre>
  * Describes a message type.
- * </pre>
  *
- * Protobuf type <code>google.protobuf.DescriptorProto</code>
+ * Protobuf type <code>Google\Protobuf\Internal</code>
```

Comment:
Could you remove the line for "Protobuf type < code>Google\Protobuf\Internal</ code>"